dist/scss/wds-components/_dropdowns.scss

```diff
@@ -17,30 +21,188 @@
+		border-radius: $border-radius;
 		display: none;
-		left: 0;
+		left: 50%;
+		min-width: 60%;
+		// we assume that there will be regular size .wds-list inside the dropdown content
+		padding: $wds-dropdown-padding - $wds-list-item-vertical-padding 0;
 		position: absolute;
-		top: calc(100% + 1px);
+		top: 100%;
+		transform: translateX(-50%);
 		z-index: $content-z-index;
 
-		&.wds-is-dark {
-			box-shadow: 0 0 20px 0 rgba($wds-color-black, .3);
-		}
-
 		&.wds-is-right-aligned {
 			left: auto;
 			right: 0;
+			transform: none;
+		}
+
+		&.wds-is-left-aligned {
+			left: 0;
+			transform: none;
+		}
+
+		// The default is to show scroll on long dropdowns
+		&:not(.wds-is-not-scrollable) {
+			.wds-list {
+				@include scrollable-list(270px);
+			}
+		}
+
+		&.wds-is-not-scrollable {
+			.wds-list {
+				position: relative;
+			}
+		}
+
+		.wds-list {
+			padding: 0;
+
+			> li {
+				margin: 0 $wds-dropdown-padding / 2;
+				padding-left: $wds-dropdown-padding / 2;
+				padding-right: $wds-dropdown-padding / 2;
+				white-space: nowrap;
+			}
+
+			&.wds-is-linked {
+				> li {
+					padding: 0;
+
+					> a {
+						color: $wds-color-dark-blue-gray;
+						padding-left: $wds-dropdown-padding / 2;
+						padding-right: $wds-dropdown-padding / 2;
+						text-decoration: none;
+						transition: color $link-transition-duration;
+					}
+
+					&:hover > a {
+						@include wds-dropdown-linked-list-item-hover;
+					}
+				}
+			}
 		}
 	}
 
-	&.wds-is-active .wds-dropdown {
-		&__content {
+	&::after {
+		border: 8px solid transparent;
+		border-bottom-color: $wds-color-white;
+		bottom: -$dropdown-border-width;
+		content: '';
+		display: none;
+		height: 0;
+		left: 50%;
+		margin-left: -8px;
+		pointer-events: none;
+		position: absolute;
+		width: 0;
+		z-index: $content-z-index + 1;
+	}
+
+	&::before {
+		border: 9px solid transparent;
+		border-bottom-color: $wds-color-faint-gray;
+		bottom: -$dropdown-border-width;
+		content: '';
+		display: none;
+		height: 0;
+		left: 50%;
+		margin-left: -9px;
+		pointer-events: none;
+		position: absolute;
+		width: 0;
+		z-index: $content-z-index + 1;
+	}
+
+	&:hover::after {
+		display: block;
+	}
+
+	&:hover::before {
+		display: block;
+	}
+
+	&.wds-is-active,
+	&:hover {
+		.wds-dropdown__content {
 			display: inline-block;
 		}
 
-		&__toggle-chevron {
+		.wds-dropdown__toggle-chevron {
 			transform: rotate(180deg);
 		}
+
+		&.wds-has-shadow, &.wds-has-dark-shadow {
+			&::before {
+				display: none;
+			}
+		}
+	}
+
+	&.wds-has-shadow {
+		.wds-dropdown__content {
+			box-shadow: 0 0 10px 0 rgba($wds-color-black, .2);
+		}
+	}
+
+	&.wds-has-dark-shadow {
+		.wds-dropdown__content {
+			box-shadow: 0 0 20px 0 rgba($wds-color-black, .3);
+		}
+	}
+
+	&.wds-has-shadow, &.wds-has-dark-shadow {
+		.wds-dropdown__content {
+			border: 0;
+		}
+	}
+}
+
+.wds-dropdown__content .wds-list.wds-is-linked .wds-dropdown-level-2 {
+	margin-right: 0;
+
+	> a {
+		padding-right: 18px;
```

Comment:
maybe we could use some calculation of existing dropdown padding/margin variables? I guess they are somehow correlated.